packages/components/src/theme/cards.scss

```diff
@@ -123,3 +123,104 @@ $circle-size: 12 * $scale;
 .cards__block-disabled {
   background: linear-gradient(0.15turn, #aaa, #eee) no-repeat center;
 }
+
+.filter-cards {
+    display: flex;
+    overflow-x: scroll;
+    padding-left: 1px;
+    padding-bottom: 20px;
+    justify-content: left;
+
+    &.empty {
+        justify-content: center;
+        margin-top: 100px;
+    }
+}
+
+.filter-cards__card {
+    //display: block;
+    min-width: 330px;
+    border-radius: 2px;
+    height: $card-min-height;
+    background: white;
+    padding: 0;
+    transition: all 250ms;
+    font-size: 14px;
+    margin-right: 15px;
+
+
+    @include shadow(1);
+
+    &:hover,&:focus {
+        text-decoration: none;
+
+        @include shadow(3);
+        transform: translateY(-5px);
+        cursor: pointer;
+
+        .cards__card-content {
+            color: #555;
+        }
+        .cards__card-title {
+            color: #333;
```

Comment:
do we have a scss variable for this gray?

Reply:
Turns out we do. Updated here and elsewhere in this file.